src/main/java/org/dcsa/ovs/controller/UnofficialTransportTOController.java

```diff
@@ -0,0 +1,34 @@
+package org.dcsa.ovs.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.dcsa.core.controller.ExtendedBaseController;
+import org.dcsa.core.extendedrequest.ExtendedRequest;
+import org.dcsa.ovs.model.transferobjects.UnofficialTransportTO;
+import org.dcsa.ovs.service.UnofficialTransportTOService;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "unofficial-transports", produces = {MediaType.APPLICATION_JSON_VALUE})
+public class UnofficialTransportTOController extends ExtendedBaseController<UnofficialTransportTOService, UnofficialTransportTO, UUID> {
```

Comment:
Why call this "UnofficialTransportTOController"? None of the other unofficial endPoint controllers have this prefix?

Reply:
No good reason. I misnamed it :)

Reply:
It is a reference to `UnofficialTransportTO` because the entity is not an official one.